indexing-service/src/main/java/io/druid/indexing/common/actions/SegmentListUsedAction.java

```diff
@@ -34,16 +37,29 @@
   private final String dataSource;
 
   @JsonIgnore
-  private final Interval interval;
+  private final List<Interval> intervals;
 
   @JsonCreator
   public SegmentListUsedAction(
```

Comment:
Might be nice to have other constructors, as this is created programmatically a lot and the `null`s are a bit weird